01_install_requirements.sh

```diff
@@ -9,14 +9,6 @@ if grep -q "Red Hat Enterprise Linux release 8" /etc/redhat-release 2>/dev/null
     RHEL8="True"
 fi
 
-sudo yum install -y libselinux-utils
-if selinuxenabled ; then
-    # FIXME ocp-doit required this so leave permissive for now
-    sudo setenforce permissive
-    sudo sed -i "s/=enforcing/=permissive/g" /etc/selinux/config
-fi
-
-
```

Comment:
For other reviewers, this is handled via https://github.com/metal3-io/metal3-dev-env/blob/master/centos_install_requirements.sh#L9..L13 which is run below L26